_includes/context/positioning.md

```diff
@@ -1,8 +1,17 @@
+{% case page.title %}
+  {% when "Context toolbar" %}
+    {% assign contextItemType = "toolbar" %}
+  {% when "Context forms" %}
+    {% assign contextItemType = "form" %}
+{% endcase %}
+<a class="anchor" id="positioningcontexttoolbars"></a><a class="anchor" id="positioningcontextforms"></a>
 
-There are three options for positioning: `selection`, `line`, or `node`.
+### Positioning of context {{contextItemType}}
```

Comment:
This is incorrect, as it should be `Positioning of context forms`, whereas atm it's `Positioning of context form`. So it should have a trailing `s` for both toolbars and forms.

Reply:
Added the `s` in https://github.com/tinymce/tinymce-docs/pull/1517/commits/343c4fd8c3d4cf158d9dc1ac98df01601665e184